include/libopencm3/stm32/f4/rcc.h

```diff
@@ -1092,6 +1092,9 @@ enum rcc_periph_rst {
 	RST_SAI1RST	= _REG_BIT(0x24, 22),/* F42x, F43x */
 	RST_LTDC	= _REG_BIT(0x24, 26),/* F42x, F43x */
 	RST_DSI		= _REG_BIT(0x24, 27),/* F42x, F43x */
+
+	/* Advanced peripherals */
```

Comment:
Where does this "advanced peripherals" come from? It's not something that seems to be in rm0090 anywhere?

Reply:
I saw it in the f0 section of the rcc so I just copy-pasted it

Reply:
I've changed the name from "Advanced peripherals" to the rm0090s "Backup domain control"